package.json

```diff
@@ -8,7 +8,10 @@
   "version": "0.0.0",
   "license": "Apache-2.0",
   "private": true,
-  "repository": "https://github.com/IBM/openapi-validator",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/IBM/openapi-validator.git"
+  },
```

Comment:
We were getting warnings in Travis that our `package.json` files were incorrectly formatted. Sure enough, NPM changed the structure a while back and we missed the memo. This commit updates to the structure recommended in their docs